test/functional/src/main.cpp

```diff
@@ -98,12 +111,11 @@ int main(int argc, char **argv)
     });
 
     // RSA APIs
+    if (isRsaConfigured)
     {
-        bool isConfigured;
         RsaTests rsaTests(logLocation);
-        RunTest("== RSA Configuration ==", [&rsaTests, &isConfigured](){return (isConfigured = rsaTests.IsConfigured());});
 
-        if (isConfigured)
+        if (RunTest("== RSA setup ==",            [&rsaTests](){return rsaTests.Setup();}))
```

Comment:
>[&rsaTests](){return rsaTests.Setup();})) [](http://example.com/codeflow?start=50&length=41)

Nit: I think you can do this without a lambda by passing in `std::bind(&RsaTests::Setup, rsaTests)`